src/main/java/cz/centrum/haffner/SimpleTrainingDavid/AppServices/KpisInfoProcessor.java

```diff
@@ -0,0 +1,46 @@
+package cz.centrum.haffner.SimpleTrainingDavid.AppServices;
+
+import cz.centrum.haffner.SimpleTrainingDavid.DataTemplates.KpisInfoData;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@Component
+public class KpisInfoProcessor {
+    private Set differentOriginsSet = new HashSet<Integer>();
+    private Set differentDestinationsSet = new HashSet<Integer>();
+
+    @Autowired
+    private KpisInfoData kpisInfoData;
+
+
+    public void addOneToProcessedFilesNumber() { kpisInfoData.addOneToProcessedFilesNumber(); }
```

Comment:
Doporučuju pro metody pro inkrementaci používat slovo increment.
Opět, než mi hlava přečte a pochopí "addOneTo", tak to trvá mnohem déle.
Druhý podstatnější důvod - ve chvíli, kdy bys chtěl aby metoda přičítala dva nebo tři nebo víc?
Budeš vytvářet metody: addTwoTo..., AddThreeTo...? 
A nebo bude lepší mít jen 2 metody ve stylu:

// Increment filedX by 1
public void incrementFieldX() {
  incrementFieldX(1)
}

// Increment fieldX by n
public void incrementFieldX(int amountToAdd) {
  (if fieldX > 0) {
     fieldX += amountToAddů;
  }
}